x-pack/test/functional/services/actions/common.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ProvidedType } from '@kbn/test';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export type ActionsCommon = ProvidedType<typeof ActionsCommonServiceProvider>;
+
+export function ActionsCommonServiceProvider({ getService, getPageObject }: FtrProviderContext) {
+  const testSubjects = getService('testSubjects');
+
+  return {
+    async clickCreateFirstConnectorButton() {
+      const createBtn = await testSubjects.find('createFirstActionButton');
+      const createBtnIsVisible = await createBtn.isDisplayed();
+      if (createBtnIsVisible) {
+        await createBtn.click();
```

Comment:
I think is better to fail if the button does not exist. You can use `testSubjets.existsOrFail`.